policyuniverse/expander_minimizer.py

```diff
@@ -171,17 +172,15 @@ def minimize_statement_actions(statement, minchars=None):
 def get_actions_from_statement(statement):
     allowed_actions = set()
 
-    if not type(statement.get("Action", [])) == list:
-        statement["Action"] = [statement["Action"]]
+    statement["Action"] = ensure_array(statement.get("Action", []))
 
-    for action in statement.get("Action", []):
+    for action in statement["Action"]:
         allowed_actions = allowed_actions.union(set(_expand_wildcard_action(action)))
 
-    if not type(statement.get("NotAction", [])) == list:
-        statement["NotAction"] = [statement["NotAction"]]
+    statement["NotAction"] = ensure_array(statement.get("NotAction", []))
 
     inverted_actions = set()
-    for action in statement.get("NotAction", []):
+    for action in statement["NotAction"]:
```

Comment:
Same question as above:

Won't this fail and throw an exception if the statement uses `Action` instead of `NotAction` ?